unicodetools/src/main/java/org/unicode/props/PropertyParsingInfo.java

```diff
@@ -48,12 +49,34 @@ enum SpecialProperty {
     public final UcdProperty property;
     public final SpecialProperty special;
 
+    public static class FieldMapping implements Comparable<FieldMapping> {
```

Comment:
please add a brief `/** class API doc */` describing the purpose/use of this new class